service/service.go

```diff
@@ -86,10 +86,10 @@ func New(config Config) (*Service, error) {
 
 	var certWatcher *certificatetpr.Service
 	{
-		certConfig := certificatetpr.DefaultConfig()
+		certConfig := certificatetpr.DefaultServiceConfig()
```

Comment:
This is for the changes introduced here: https://github.com/giantswarm/certificatetpr/pull/23